sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
+## 1.1.0-beta.1 (2021-07-01)
```

Comment:
Add an entry for the new feature, MSI Credential

Reply:
ManagedIdentityCredential PR adds it.

Reply:
See just below now.